### ISSUE TITLE
Still issues with Posten?

### ISSUE THREAD

**Issue by tbarmo** [reidarw/MMM-PostDelivery-Norway#6]
It has been a while since this module worked for me. Tried to reinstall the module, but it is still not loading.

Is this module dead because of changes Posten does on their end?

**Comment by reidarw** [comment]
Strange. It works for me and has been stable since my last fix (5 months ago).
My settings:
```json
        {
            module: 'MMM-PostDelivery-Norway',
            position: 'top_left',
            config: {
                zipCode: 7033,
                numberOfDays: 2,
                header: "Postlevering"
            }
        },
```

**Comment by tbarmo** [comment]
Thanks for the reply. 

It worked for a while after your last update. Now it just says "Laster...". The settings was not changed from when it worked to it didn't work anymore. All the other modules I use are working.

I did a new installation of MM yesterday on another Pi with a new sd-card. It still says "Laster...". From what I can read from your source code, something failed or it is waiting for something (line 50 and 51 in your code).

**Comment by reidarw** [comment]
Can you post your settings, then I can test them when I find time

**Comment by tbarmo** [comment]
Here it is. I testet also connecting the Pi to ethernet in case there was som issues with my wireless. Still the same. All modules except yours starts properly.  In the screenshot on the right side it just says "Laster..." My configuration is not very complicated either. 

Thanks 

Config file here: [config.js](https://github.com/reidarw/MMM-PostDelivery-Norway/files/12612639/config.txt)

**Comment by tbarmo** [comment]
I did some troubleshooting now and this is what I found out.  
- I commented out all other modules than yours. Now it presents itself properly. 
- I introduced one module at a time with a restart of MM in between. 
- Your module worked all the time, until I uncommented the module MMM-GoogleMapsTraffic, the last one in my configuration. 

It appears that there are some compatibility issue with the GoogleMapsTraffic module and your module. With only GoogleMapsTraffic and PostDelivery module enebled, your module show only "Laster...". Removing GoogleMapsTraffic from the config and your module is working once more.

I hope this information helps.

**Comment by tbarmo** [comment]
One last post here. It seems there is a (known) problem with the GoogleMapsTraffic module and other modules (your included).

The solution is in this post in the MM forum:
[https://forum.magicmirror.builders/topic/12002/mmm-googlemapstraffic-module-stops-mmm-mysportsboard-from-working](url)

I downloaded the modified version of the GoogleMapsTraffic, and it fixed the issue I had with your module.

Closing this thread now.
